dracut/99emergency-timeout/timeout.sh

```diff
@@ -29,7 +29,7 @@ _prompt_for_timeout() {
     if [[ -e /.emergency-shell-confirmed ]]; then
         return
     fi
-    ignition_units="ignition-disks.service ignition-files.service ignition-mount.service"
+    ignition_units="ignition-fetch.service ignition-disks.service ignition-files.service ignition-mount.service"
```

Comment:
I wonder if we can just glob on `ignition-*` here so we don't hit this problem in the future if we add more ignition units.